Sources/Navigation/Extensions/WKFrameInfoExtension.swift

```diff
@@ -58,6 +58,8 @@ public extension WKFrameInfo {
 
         // ignore `request` selector calls from `safeRequest` itself
         ignoredRequestUsageSymbols.insert(callingSymbol())
+        // ignore `-[WKFrameInfo description]`
+        ignoredRequestUsageSymbols.insert("-[WKFrameInfo description]")
```

Comment:
fix debug Breakpoint hitting when running `po navigationAction` or `po frameInfo` in debugger